code/modules/events/shuttle_loan.dm

```diff
@@ -220,10 +234,36 @@
 			var/spawn_type = pick_n_take(shuttle_spawns)
 			new spawn_type(T)
 
+//items that appear only in shuttle loan events
+
+/obj/item/storage/belt/fannypack/yellow/bee_terrorist/PopulateContents()
+	new /obj/item/grenade/plastic/c4 (src)
+	new /obj/item/reagent_containers/pill/cyanide(src)
+	new /obj/item/grenade/chem_grenade/facid(src)
+
+/obj/item/paper/fluff/bee_objectives
+	name = "Objectives of a Bee Liberation Front Operative"
+	info = "<b>Objective #1</b>. Liberate all bees on the NT transport vessel 2416/B. <b>Success!</b>  <br><b>Objective #2</b>. Escape alive. <b>Failed.</b>"
+
+/obj/machinery/syndicatebomb/shuttle_loan/Initialize()
+	. = ..()
+	setAnchored(TRUE)
```

Comment:
Given that bomb defusal is pure RNG, forcing the shuttle to explode like that doesn't really seem kosher. At least unachor  it

Reply:
You can refuse the deal if you don't want to risk it, and this means you can just get free cash by taking the shuttle, throwing the bomb out the airlock, then sending it back.

Reply:
Making the bomb unanchored would make the fact it's there in the first place redundant, might aswell call it free points